### PR TITLE
KATI-88_DoctorSettingFunction

### PR DESCRIPTION
**Pull request by toannguyen71195**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/activity/SettingEditWorkingHourActivity.java [hunk 1]
```diff
@@ -172,18 +189,67 @@ public void showEndTimePicker() {
...
+            if (elem.equals(s)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private int roundMinute(int minute) {
+        if (minute > 12 && minute < 48) {
+            return 30;
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/SettingEditWorkingHourActivity.java]: What is 12, 48 and 30 ? Should make them constants with meaningful names.

File: app/src/main/java/com/kms/cura/view/activity/SettingEditWorkingHourActivity.java [hunk 2]
```diff
@@ -172,18 +189,67 @@ public void showEndTimePicker() {
...
+        }
+        return 0;
+    }
+
+    private int roundHour(int hour, int minute) {
+        if (minute > 47) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/SettingEditWorkingHourActivity.java]: Same here with 47

File: app/src/main/java/com/kms/cura/view/activity/SettingWorkingHourActivity.java [hunk 3]
```diff
@@ -186,4 +193,105 @@ public void removeItem(String facility, OpeningHour openingHour) {
...
+                    exception = e;
+                }
+                return null;
+            }
+
+            @Override
+            protected void onPostExecute(Void aVoid) {
+                pDialog.dismiss();
+                if (exception != null) {
+                    ErrorController.showDialog(SettingWorkingHourActivity.this, "Error : " + exception.getMessage());
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/SettingWorkingHourActivity.java:260]: return after this to remove below else

File: app/src/main/java/com/kms/cura/view/activity/SettingEditWorkingHourActivity.java [hunk 4]
```diff
@@ -40,6 +40,9 @@
     public static final String EDIT_WORKING_HOUR_POSITION = "edit_position";
     public static final int UPDATE_FACILITY_CODE = 2;
     private boolean endDateSet = false;
+    private final int lowerBound = 12;
+    private final int upperBound = 47;
+    private final int midBound = 30;
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/SettingEditWorkingHourActivity.java]: Make these fields constant, and name them with all uppercase

**Comment by hieumai** [comment]
OK from me. @vtnlinh95 please start verifying this

**Comment by vtnlinh95** [comment]
@toannguyen71195  : 
- Other can not edit
- After editting something, the password of Current User is wrong so can not change password

**Comment by hieumai** [comment]
OK from me. @toannguyen71195  please resolve conflicts so I can merge

**Pull request merged**